plugin.json

```diff
@@ -15,27 +15,27 @@
         "footer": "",
         "settings": [{
             "key": "TeamChannel",
-            "display_name": "Team/Channel",
+            "display_name": "Channel to send notifications to:",
             "type": "text",
-            "help_text": "Name of the Team and Channel you want to create/use to receive the SNS messages. Split by comma. ie. teamA,channelXYZ"
+            "help_text": "The channel to send notifications to, specified in the format `teamname,channelname`. If the specified channel does not exist, the plugin will create the channel for you.\n\nNote: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel"
         },
         {
             "key": "AllowedUserIds",
-            "display_name": "AllowedUserIds",
+            "display_name": "Authorized users who can accept AWS SNS subscriptions:",
```

Comment:
Not in favor of this setting name. Would prefer something shorter (any ideas?)

Reply:
None other than just User IDs

Reply:
Switched it back to `Authorized User IDs`